src/main/java/org/tarantool/TarantoolClusterClient.java

```diff
@@ -194,13 +199,18 @@ protected void onReconnect() {
             // First call is before the constructor finished. Skip it.
             return;
         }
-        Collection<TarantoolOp<?>> futuresToRetry = new ArrayList<>(retries.values());
+        Collection<TarantoolOp<?>> delayed = new ArrayList<>(retries.values());
+        Collection<TarantoolOp<?>> reissued = new ArrayList<>(retries.size());
         retries.clear();
-        for (final TarantoolOp<?> future : futuresToRetry) {
+        for (final TarantoolOp<?> future : delayed) {
             if (!future.isDone()) {
                 executor.execute(() -> registerOperation(future));
+                reissued.add(future);
             }
         }
+        for (final TarantoolOp<?> future : reissued) {
+            LOGGER.trace("{0} was re-issued after reconnection", future);
```

Comment:
I failed to get the delayed / re-issued messages on tests. How can I check how they actually looks?

Reply:
It's hard to catch and probable that wasn't tested. So, I added a new synt test where the retries are involved

https://github.com/tarantool/tarantool-java/blob/a095ee2306c7e273a781333a6a7b6f8ee7d037e7/src/test/java/org/tarantool/ClientReconnectClusterIT.java#L87-L89

Reply:
Nice! I got it:

> org.tarantool.TarantoolClusterClient Request TarantoolOp{id=2, code=EVAL, args=[EXPRESSION, return 1+2, TUPLE, [Ljava.lang.Object;@63a72cc6]} was delayed because of org.tarantool.CommunicationException: Cant read answer 

The only question here: What is `Ljava.lang.Object;@63a72cc6`? Should not arguments list that is passed to the lua closure be empty in case of `client.asyncOps().eval("return 1+1")` (yep, AFAIR you can pass arguments for eval and use them with `...`)? Should not we try to serialize arguments instead of printing of their addresses (at least for POD)?

Reply:
I refactored this in #200, so it's required to be adapted to the new format later. Moreover, if we can pass different objects as arguments we cannot guarantee that all of them have an appropriate `toString()` implementation. I think It is worth skipping args part and, say, printing out a number of arguments, instead.

The `Ljava.lang.Object;@63a72cc6` is what we see each time when `toString()` is not overridden and Object.toString() is invoked.

Reply:
Okay, now I see: this object is a list of arguments. I think it is okay to hide arguments for now and maybe add it to log messages later.